documentation/high-level-introduction.md

```diff
@@ -0,0 +1,251 @@
+# A brief introduction to msbuild (Level 101)
+
+## Introduction
+
+Documentation on msbuild is both widespread and yet much of it is difficult to parse for those coming up to speed. There are also no good "high level" docs that paint a picture on how the parts are put together.
```

Comment:
I would probably move this justification into the PR comment, the reader doesn't need to know